timescaledb/how-to-guides/install-timescaledb/installation-debian.md

```diff
@@ -0,0 +1,161 @@
+# Install self-hosted TimescaleDB on Debian-based systems
+You can host TimescaleDB yourself, on your Debian or Ubuntu system. These
+instructions use the `apt` package manager on these distributions:
+
+*   Debian 9 Stretch
+*   Debian 10 Buster
+*   Debian 11 Bullseye
+*   Ubuntu 18.04 LTS Bionic Beaver
+*   Ubuntu 20.04 LTS Focal Fossa
+*   Ubuntu 21.04 Hirsute Hippo
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `apt`
+package manager, you could encounter errors following these instructions. It is
+safest to remove any existing PostgreSQL installations before you begin. If you
+want to keep your current PostgreSQL installation, do not install TimescaleDB
+TimescaleDB using this method.
+[Install from source](/how-to-guides/install-timescaledb/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on Debian-based systems
+1.  At the command prompt, as root, add the PostgreSQL third party repository
```

Comment:
We say 'as root' here, but then the next command (at least for Ubuntu) we have them use `sudo`. My guess is that it was different in the Debian vs. Ubuntu original docs. Generally, I think most docs assume folks are not root anymore and lean towards `sudo`?

Reply:
Ah, that's my bad. We shouldn't be using sudo. It's actually because of my testing environments, I was using both methods.